sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session.py

```diff
@@ -110,6 +110,7 @@ def get_state(self, *, timeout: Optional[float] = None, **kwargs: Any) -> bytes:
         """
         if kwargs:
             warnings.warn(f"Unsupported keyword args: {kwargs}")
+        self._receiver: ServiceBusReceiver
```

Comment:
is there an assignment or some other operation missing?

Reply:
If we improve the typing on `BaseSession` we might not have to do this (but it doesn't hurt)